pkg/api/operators/apis/trialmetrics/v1alpha2/trialmetrics_types.go

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
is this meant to store final metrics of Trials?

Reply:
I had thought to use it persist best metrics by interval from DB. but per discuss, maybe we can drop it and trial controller will get final metrics to fill trial status when worker done